### DISCUSSION TITLE
Custom Ingress Resource Health

### DISCUSSION THREAD

**Discussion by oyuskeseliev** [argoproj/argo-cd#6745] [Q&A] [answered]
Hi all,

I`m trying to find documentation regarding Custom Resource Health checks more specifically for Ingress (Traefik and Contour). Sadly I do not see a lot of info how to handle such a custom config. Dose someone have experience and can point me to the correct documentation or maybe shared some feedback on example implementations? 

Regards,
Ognyan

**Comment by jgwest** [comment, marked as answer]
Hi @oyuskeseliev, if you haven't already seen them, you can check out the general resource [health customization page](https://argo-cd.readthedocs.io/en/stable/operator-manual/health/) and a bunch of [existing, built-in resource customizations](https://github.com/argoproj/argo-cd/tree/master/resource_customizations) as examples, but I don't see any that are specifically for Ingress/Traefik/Contor (though certainly some around managing certs).

Also see [this bug](https://github.com/argoproj/argo-cd/issues/968), **NOTE: it's from 2019 😄**, but it talks about issues around how Traefik/Contour handle Ingresses which might provide some helpful context.

**Reply by oyuskeseliev** [reply]
Hi @jgwest ,

thanks for the resources. In the mean time I saw that Traefik 2.5 will be migrating to networking.k8s.io/v1 so I hope that this will solve the problem or I will use the workaround described [here](https://github.com/traefik/traefik/issues/6303).

Thanks,
Ognyan
